src/linking/module_dependencies.txt

```diff
@@ -1,3 +1,4 @@
+ansi-c
```

Comment:
❓ Isn't linking a pure GOTO thing? Why is there a dependency on ansi-c now? This seems to indicate that either some functionality has been moved that should not be moved or that some functionality is used in other modules that shouldn't be used there.

Reply:
The culprit for this is `recreate_initialize_function`. I'd like to kill it, and one benefit would be to remove this coupling.

Reply:
Being the one who introduced this function in the first place: can you share some ideas on how to replace it?

Reply:
As mentioned out-of-band, simply create the goto instructions directly. This should be done separately.